src/Kliva/Views/MainPage.xaml.cs

```diff
@@ -24,7 +24,6 @@ public MainPage()
         protected override void OnNavigatedTo(NavigationEventArgs e)
         {
             base.OnNavigatedTo(e);
-            UpdateVisualState(VisualStateGroup.CurrentState);
             _activityList.SelectedIndex = -1;
```

Comment:
Yes it seems like it won't be necessary. Removed this from MainPage and ClubsPage.